.hlint.yaml

```diff
@@ -7,7 +7,6 @@
 
 
 # Warnings currently triggered by your code
-- ignore: {name: "Parse error"}
```

Comment:
That's probably not something you want to be ignoring?

Reply:
Absolutely, this is the reason I removed it 😉 

The reason it was there before was that hlint was not able to parse Typed TH and could not be run on those files. But as hlint supports Typed TH nowadays, I wanted to remove it.

Reply:
Oh, doh. I missed that it was red, not green.